### PR TITLE
fixes #1501 -> maybe i am missing more parameters where on/off is app…

### LINKED ISSUE #1501: WebUI - Slider behavior for "Switches"
In the Parameter Tab, editing switch-like Parameters (Osc A/B Reset, Mono Enable) feels a little exhausting, as the user has to drag over 70% of the Slider before changes become effective. (It could be more convenient, if the Slider is more sensitive when editing switches).

### PR DESCRIPTION
**Pull request by schlenkibus**
…licable?

### CODE REVIEW

File: projects/playground/src/parameters/ValueRange.h [hunk 1]
```diff
@@ -36,6 +36,11 @@ template <typename tValue> class ValueRange
     return m_min == -m_max;
   }
 
+  bool isBoolean() const
```
Comment (hhoegelo) [inline projects/playground/src/parameters/ValueRange.h:39]: Hmmmm, I don't get his.

File: projects/NonMaps/src/main/java/com/nonlinearlabs/client/dataModel/ValueUpdater.java [hunk 2]
```diff
@@ -18,6 +18,12 @@ public void doUpdate() {
 		String bipolar = getChildText(root, "bipolar");
 		String scaling = getChildText(root, "scaling");
 		String defaultValue = getChildText(root, "default");
+		String isBoolean = getChildText(root, "boolean");
```
Comment (hhoegelo) [inline projects/NonMaps/src/main/java/com/nonlinearlabs/client/dataModel/ValueUpdater.java:21]: Couldn't we deduce the 'booleaness' from the number of steps?

File: projects/playground/src/parameters/scale-converters/EnvelopeAttackDecayTimeMSScaleConverter.cpp [hunk 3]
```diff
@@ -43,9 +43,4 @@ Glib::ustring EnvelopeAttackDecayTimeMSScaleConverter::controlPositionToDisplayJ
 bool EnvelopeAttackDecayTimeMSScaleConverter::isBiPolar() const
```
Comment (hhoegelo) [inline projects/playground/src/parameters/scale-converters/EnvelopeAttackDecayTimeMSScaleConverter.cpp:43]: Is this method still needed?

File: projects/playground/src/parameters/value/RawValue.h [hunk 4]
```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "playground.h"
+#include <optional>
```
Comment (hhoegelo) [inline projects/playground/src/parameters/value/RawValue.h:4]: can prolly be removed?

File: projects/playground/src/parameters/value/RawValue.cpp [hunk 5]
```diff
@@ -79,12 +79,12 @@ bool RawValue::isBiPolar() const
 
 void RawValue::setIsBoolean(bool v)
```
Comment (hhoegelo) [inline projects/playground/src/parameters/value/RawValue.cpp:80]: are this methods used?

Comment (schlenkibus) [reply projects/playground/src/parameters/value/RawValue.cpp:80]: The Router Parameters use this function

**Pull request merged**